<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>anatta</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabExport">
       <attribute name="title">
        <string>数据</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableViewFemale">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>统计</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tabConfig">
       <attribute name="title">
        <string>设置</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>260</y>
          <width>501</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>照片存放路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditImagePath"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonImagePath">
           <property name="text">
            <string>证件照路径</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>230</y>
          <width>501</width>
          <height>26</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>法会信息：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditFahui_name">
           <property name="placeholderText">
            <string>法会名称</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditLastMaleCode">
           <property name="placeholderText">
            <string>上次男众最大皈依号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditLastFemaleCode">
           <property name="placeholderText">
            <string>上次女众最大皈依号</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>290</y>
          <width>501</width>
          <height>28</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>数据备份路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditBackPath"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonBackPath">
           <property name="text">
            <string>数据备份路径</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>170</y>
          <width>447</width>
          <height>48</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>注意：法会信息要提前设定，主要是为了皈依证号的生成，所以千万不能写错</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>           照片存放路径是RSYNC服务的文件夹，默认是家路径下的 RsyncShare</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1017</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionSetting"/>
    <addaction name="actionExportPdf"/>
    <addaction name="action"/>
    <addaction name="actionCard"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFILE">
    <property name="title">
     <string>FILE</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menuFILE"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDb"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionExcel"/>
   <addaction name="actionPdf"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);</string>
   </property>
  </widget>
  <action name="actionSetting">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionExportPdf">
   <property name="text">
    <string>导出PDF</string>
   </property>
  </action>
  <action name="actionDb">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/db.png</normaloff>:/pics/db.png</iconset>
   </property>
   <property name="text">
    <string>db</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/see.png</normaloff>:/pics/see.png</iconset>
   </property>
   <property name="text">
    <string>refresh</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>导出数据库文件</string>
   </property>
   <property name="toolTip">
    <string>exportSql</string>
   </property>
  </action>
  <action name="actionExportPersonalInfo">
   <property name="text">
    <string>导出个人信息</string>
   </property>
  </action>
  <action name="actionPrintPersonnelCredentials">
   <property name="text">
    <string>导出个人皈依证</string>
   </property>
  </action>
  <action name="actionCard">
   <property name="text">
    <string>导出证件</string>
   </property>
  </action>
  <action name="actionExcel">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/excel.png</normaloff>:/pics/excel.png</iconset>
   </property>
   <property name="text">
    <string>excel</string>
   </property>
  </action>
  <action name="actionPdf">
   <property name="icon">
    <iconset resource="pics.qrc">
     <normaloff>:/pics/pdf.png</normaloff>:/pics/pdf.png</iconset>
   </property>
   <property name="text">
    <string>pdf</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pics.qrc"/>
 </resources>
 <connections/>
</ui>
